manifests/env.pp

```diff
@@ -18,7 +18,6 @@
 
   # Service configuration options
   $service_name     = 'strongswan'
-  $service_provider = 'upstart'
```

Comment:
Instead  - just set it to `undef` and leave the setting here. This code was added to help support other OS's I think by https://github.com/Nextdoor/puppet-strongswan/pull/9.

Reply:
👍